bake/bake_test.go

```diff
@@ -2019,6 +2019,24 @@ target "app" {
 	})
 }
 
+// https://github.com/docker/buildx/pull/428
+// https://github.com/docker/buildx/issues/2822
+func TestEmptyAttribute(t *testing.T) {
+	fp := File{
+		Name: "docker-bake.hcl",
+		Data: []byte(`
+target "app" {
+  output = [""]
+}
+`),
+	}
+
+	ctx := context.TODO()
+
+	_, _, err := ReadTargets(ctx, []File{fp}, []string{"app"}, nil, nil)
+	require.NoError(t, err)
+}
```

Comment:
This test works in current state but when looking at the `app` target it contains an empty `ExportEntry`, while previously there were none. So I guess we need to make some changes in the HCL parser as well.

Reply:
It fails as expected: https://github.com/docker/buildx/actions/runs/12120057362/job/33787924177?pr=2832#step:6:1326

```
=== FAIL: bake TestEmptyAttribute (0.00s)
    bake_test.go:2038: 
        	Error Trace:	/home/runner/work/buildx/buildx/bake/bake_test.go:2038
        	Error:      	"[]" should have 0 item(s), but has 1
        	Test:       	TestEmptyAttribute
```